mars/tensor/arithmetic/fmax.py

```diff
@@ -27,6 +27,9 @@ class TensorFMax(TensorBinOp):
     _op_type_ = OperandDef.FMAX
     _func_name = 'fmax'
 
+    def jax_function(self):
```

Comment:
Can we move it into the base class so we do not need to return the same stuff again and again? What's more, it seems more pythonic to return ``NotImplemented`` rather than ``NotImplementedError``.